nbconvert/filters/tests/test_markdown.py

```diff
@@ -137,6 +137,18 @@ def test_markdown2html_math(self):
             "$$a<b&b<lt$$",
             "$$a<b&lt;b>a;a-b<0$$",
             "$$<k'>$$",
+            """$$x
+=
+2$$""",
+            r"""$$
```

Comment:
Does this example need to be so complicated? Please don't use an example that requires a raw literal string to be legible.

Reply:
What is this example adding over the previous examples?